Felienne/hedy
# [BUG FIXES] Fixes several issues related to mail templates

**Description**
In this PR we fix several issues related to mail templates. The following things are fixes:

- `body_plain` was not initialized resulting in an error (action was performed)
- `response.responseText` doesn't exist on success -> jsonify the response

**Fixes**
This PR fixes #2148 fixes #2149 and fixes #2150

**How to test**
Unable to test correctly locally. Deploy to Alpha and verify that everything is worked as expected while keeping track of the Heroku logs.

**Checklist**
Done? Check if you have it all in place using this list*
  

* If you're unsure about any of these, don't hesitate to ask. We're here to help!
